assets/app/view/game/part/city_slot.rb

```diff
@@ -140,10 +140,11 @@ def on_click(event)
             # If there's a choice of tokens of different types show the selector, otherwise just place
             next_tokens = step.available_tokens(entity)
             if next_tokens.size == 1 && actions.include?('place_token')
+              token_owner = step.respond_to?(:token_owner) && step.token_owner(@selected_company || @game.current_entity)
```

Comment:
put this method in game and don't use respond to

Reply:
OK